@xen-orchestra/lite/src/views/host/HostConsoleView.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div :class="{ 'no-ui': !uiStore.hasUi }" class="host-console-view">
+    <div v-if="hasError">{{ $t('error-occurred') }}</div>
+    <UiSpinner v-else-if="!isReady" class="spinner" />
+    <UiStatusPanel v-else-if="!isHostRunning" :image-source="monitor" :title="$t('power-on-host-for-console')" />
+    <template v-else-if="host && hostConsole">
+      <VtsLayoutConsole>
+        <RemoteConsole
+          ref="consoleElement"
+          :is-console-available="isConsoleAvailable"
+          :location="hostConsole.location"
+          class="remote-console"
+        />
+        <template #actions>
+          <VtsActionsConsole
+            :open-in-new-tab="openInNewTab"
+            :send-ctrl-alt-del="sendCtrlAltDel"
+            :toggle-full-screen="toggleFullScreen"
+            :is-fullscreen="!uiStore.hasUi"
+          />
+          <VtsDivider type="stretch" />
+          <VtsClipboardConsole />
+        </template>
+      </VtsLayoutConsole>
+    </template>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import monitor from '@/assets/monitor.svg'
+import RemoteConsole from '@/components/RemoteConsole.vue'
+import UiSpinner from '@/components/ui/UiSpinner.vue'
+import UiStatusPanel from '@/components/ui/UiStatusPanel.vue'
+import { isHostOperationPending } from '@/libs/host'
+import { HOST_OPERATION } from '@/libs/xen-api/xen-api.enums'
+import type { XenApiHost } from '@/libs/xen-api/xen-api.types'
+import { usePageTitleStore } from '@/stores/page-title.store'
+import { useConsoleStore } from '@/stores/xen-api/console.store'
+import { useControlDomainStore } from '@/stores/xen-api/control-domain.store'
+import { useHostStore } from '@/stores/xen-api/host.store'
+import VtsActionsConsole from '@core/components/console/VtsActionsConsole.vue'
+import VtsClipboardConsole from '@core/components/console/VtsClipboardConsole.vue'
+import VtsLayoutConsole from '@core/components/console/VtsLayoutConsole.vue'
+import VtsDivider from '@core/components/divider/VtsDivider.vue'
+import { useUiStore } from '@core/stores/ui.store'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useRoute, useRouter } from 'vue-router'
+
+const STOP_OPERATIONS = [HOST_OPERATION.SHUTDOWN]
+
+usePageTitleStore().setTitle(useI18n().t('console'))
+
+const router = useRouter()
+const route = useRoute()
+const uiStore = useUiStore()
+
+const {
+  isReady: isHostReady,
+  getByUuid: getHostByUuid,
+  hasError: hasHostError,
+  runningHosts,
+} = useHostStore().subscribe()
+const { records: getControlDomains } = useControlDomainStore().subscribe()
```

Comment:
related to this [comment](https://github.com/vatesfr/xen-orchestra/pull/8136#discussion_r1856675124)
```suggestion
const { records: controlDomains } = useControlDomainStore().subscribe()
```